src/controllers/user.js

```diff
@@ -1,21 +1,22 @@
 const User = require('../../models/User')
 const { StatusCodes } = require('http-status-codes')
 const { BadRequestError, UnauthenticatedError } = require('../../errors')
+const addressToCoordinate = require('../util/addressToCoordinate')
 
 const signUp = async (req, res) => {
     const user = await User.create({ ...req.body })
     const token = user.createJWT()
+    const userAddress = user.address
+    const address = await addressToCoordinate(userAddress)
```

Comment:
This should be done before creating the new user (user req.body.address). Then longitude and latitude should be passed into the user creation call.